x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -50,7 +50,13 @@ type HostIsolationExceptionPaginatedContent = PaginatedContentProps<
   typeof ExceptionItem
 >;
 
-/* eslint-disable complexity */
+const getPaginationObject = (total?: number, perPage?: number, page?: number) => ({
```

Comment:
I would strongly suggest we use object params here since more than one type is identical.

Reply:
Done in: https://github.com/elastic/kibana/pull/124167/commits/ee51496476445f04ccd01013b34c0c1724ebdebf